TTS/trainer.py

```diff
@@ -591,7 +591,17 @@ def _optimize(
 
         step_start_time = time.time()
         # zero-out optimizer
-        optimizer.zero_grad()
+        if self.model.capacitron_vae:
```

Comment:
Capacitron works with 2 optimizers. One is the main RAdam optimising all model parameters except a single param, `beta` that's used as a langrange multiplier-like constant to control the variatonal embedding. I'm utilising the `optimizer_idx` feature in the Trainer API, however because there's no need for different training steps as with the GAN models, I needed to implement a couple hacks here and there.

Reply:
While normally there's only a single optimizer being passed into this fn. in the Trainer, I need to pass both optimizers of Capacitron into here for the optimization routine shown below. Hence, I'm actually passing in a list of optimizers here and splitting them accordingly.

Reply:
Some tests are failing because they exit saying the model doesn't have this attribute. Should I rather use `if hasattr(self.model, "capacitron_vae")`?

Reply:
For reference, here's the [old training script (`train_tacotron.py`) from March that worked](https://github.com/a-froghyar/Capacitron/blob/dev-capacitron-dev/TTS/bin/train_tacotron.py).